docs/plugin-protocol/README.md

```diff
@@ -211,3 +211,27 @@ new file in this directory for each new minor version and consider all
 previously-tagged definitions as immutable. The outdated comments in those
 files are retained in order to keep the promise of immutability, even though
 it is now incorrect.
+
+## Updating the plugin protocol in Terraform core
+
+> Note: This section's audience is contributors to Terraform, not developers creating a new SDK.
+
+New features added to Terraform core often require an update to the plugin protocol. These changes are reflected
+in a new minor version of the plugin protocol. 
+
+When creating a new minor version, follow this process:
+
+1) Duplicate the files for the latest minor versions of protocol 5 and 6 and rename them to the next minor version.
```

Comment:
We should probably mention this should only be done once per Terraform minor release. We can accumulate multiple changes into a single minor version update.

Reply:
Good point- I'll update the PR with that

Reply:
Addressed in [fb99790](https://github.com/hashicorp/terraform/pull/36592/commits/fb99790e16ded7c8910a2296c48566354fb191b0)